requirements/base.txt

```diff
@@ -8,7 +8,7 @@ django-localflavor==1.2
 django-endless-pagination==2.0
 django-dotenv==1.4.1
 djangohelpers==0.18.1
-djorm-ext-pgfulltext==0.9.3
+git+git://github.com/linuxlewis/djorm-ext-pgfulltext@tags/0.9.3
```

Comment:
Shoot, sorry -- didn't mean to include this.  (I wasn't able to find this requirement unless I updated it to point directly at the github version.  But still didn't intend to put it in this commit...)